hip/include/vecmem/memory/hip/managed_memory_resource.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * VecMem project, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2024 CERN for the benefit of the ACTS project
```

Comment:
There is no one true way with this, but in this project we set the "starting date" in the copyright statements to be the year when the file was created. (Of course when we rename, or largely copy a file, the definition is more gray that this...)

So in this case, I'd vote to just have `2024` here by itself. :wink: